redis/src/connection.rs

```diff
@@ -40,13 +40,19 @@ use crate::PushInfo;
 use rustls_native_certs::load_native_certs;
 
 #[cfg(feature = "tls-rustls")]
-use crate::tls::TlsConnParams;
+use crate::tls::ClientTlsParams;
 
 // Non-exhaustive to prevent construction outside this crate
-#[cfg(not(feature = "tls-rustls"))]
-#[derive(Clone, Debug)]
+#[derive(Clone, Debug, Default)]
```

Comment:
since this is about security, I'd prefer that the values here be always explicitly set - please remove the `Default` derivation.

Reply:
:+1: